packages/blockchain-extension/extension/commands/deleteIdentityCommand.ts

```diff
@@ -26,8 +25,9 @@ export async function deleteIdentity(treeItem: IdentityTreeItem): Promise<void>
     const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
     outputAdapter.log(LogType.INFO, undefined, `deleteIdentity`);
 
-    let walletPath: string;
+    let wallet: IFabricWallet;
```

Comment:
This is an IFabricWallet as it could be either a 1.4 or 2 wallet?

Reply:
Nevermind, just remembered we're going to have separate codebases for 1.4 and 2!